gltext.h

```diff
@@ -3,12 +3,13 @@
 // License: https://github.com/MrVallentin/glText/blob/master/LICENSE.md
 //
 // Date Created: September 24, 2013
-// Last Modified: October 30, 2018
+// Last Modified: November 03, 2019
```

Comment:
For future reference, no need to modify these. They should probably just be deleted all together.